<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMacWid</class>
 <widget class="QWidget" name="CreateMacWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>MAC地址生成</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>客户名称：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="typeBox">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>自动识别</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IP-PDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ZPDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ATS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设备类型：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>90</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="snEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createBtn">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>210</y>
        <width>180</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>创 建 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="revokeBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>400</y>
        <width>140</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string>撤 销</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="macEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>MAC地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="copyBtn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>390</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>起始地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="endEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>结束地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="countEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>350</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>剩余数量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>状态信息：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipEdit">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>140</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.163</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>140</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设备地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="snCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>85</width>
        <height>30</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>设备SN：</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
